isyfact-standards-doc/src/docs/antora/modules/changelog/pages/migrationsleitfaden.adoc

```diff
@@ -24,6 +24,67 @@ Die auf dieser Seite aufgeführten Hinweise sollen IsyFact Anwendungsentwicklern
 [[kapitel-task-scheduler]]
 === Task Scheduler
 
+In IsyFact 3.0 wurden mit der Umstellung auf `isy-security` die Nutzungsvorgaben zur xref:isy-task:nutzungsvorgaben/master.adoc#spring-konfiguration[Authentifizierung von Tasks] aktualisiert.
+Statt Benutzer, Passwort und BHKNZ (siehe <<task-properties-isy-sicherheit>>) muss in den Task Properties eine `registration-id` konfiguriert werden (siehe <<task-properties-isy-security>>).
+
+[[task-properties-isy-sicherheit]]
+.Properties mit isy-sicherheit in IsyFact 2
+[source,properties]
+----
+isy.task.tasks.halloWeltTask.benutzer=TestUser
+isy.task.tasks.halloWeltTask.passwort=TestPasswort
+isy.task.tasks.halloWeltTask.bhkz=TestBHKNZ
+
+# [...]
+----
+
+[[task-properties-isy-security]]
+.Properties mit isy-security in IsyFact 3
+[source,properties]
+----
+isy.task.tasks.halloWeltTask.registration-id=TestClient
+
+# [...]
+----
+
+Eine entsprechende Spring Security ClientRegistration wird durch `isy-security` xref:isy-security:nutzungsvorgaben/master.adoc#konfigurationsparameter[ Konfigurationsparameter] bereitgestellt.
+Die folgenden Beispiele zeigen wie Konfigurationen nach der Migration aussehen können.
+Für die Task Scheduler konfiguration ist nicht relevant, ob die ClientRegistration den Resource-Owner-Password-Credentials Flow (`authorization-grant-type: password`) oder nach der empfohlenen Migration von technischen Nutzern zu Confidential Clients den Client-Credentials Flow (`authorization-grant-type: client_credentials`) nutzt.
+
+.Properties mit technischem Nutzer
+[source,properties]
+----
+isy.task.tasks.halloWeltTask.registration-id=TestClient
+
+# [...]
+
+spring.security.oauth2.client.provider.TestRealm.issuer-uri=http://localhost:9095/auth/realms/testrealm
+spring.security.oauth2.client.registration.TestClient.client-id=test-client-id
+spring.security.oauth2.client.registration.TestClient.client-secret=test-client-secret
+spring.security.oauth2.client.registration.TestClient.authorization-grant-type=password
+spring.security.oauth2.client.registration.TestClient.provider=TestRealm
+
+# Benutzer, Passwort und BHKNZ werden auf die selbe registration-id abgebildet
+
+isy.security.oauth2.client.registration.TestClient.username=TestUser
+isy.security.oauth2.client.registration.TestClient.password=TestPasswort
+isy.security.oauth2.client.registration.TestClient.bhknz=TestBHKNZ
+----
+
+.Properties nach Umstellung von technischem Nutzer auf Confidential Client
```

Comment:
```suggestion
.Properties nach Umstellung von _technischem Nutzer_ auf _technischen Client_
```

The classification _Confidential Client_ is used for applications with a trust level to keep their own authentication data in local configurations secure enough in production environments. It is no wording to classify the different users _authentication data_.